app/bundles/AssetBundle/EventListener/DetermineWinnerSubscriber.php

```diff
@@ -1,37 +1,66 @@
 <?php
 
 /*
- * @copyright   2014 Mautic Contributors. All rights reserved
+ * @copyright   2019 Mautic Contributors. All rights reserved
```

Comment:
Do not change this for existing files. It should be the current year only for newly created files.
```suggestion
 * @copyright   2014 Mautic Contributors. All rights reserved
```

Reply:
This is a new file, but it is based on an old one, so github sees it as a change.